tensorrt_llm/executor/base_worker.py

```
@@ -715,20 +715,112 @@ def _check_sleep_wakeup_preconditions(self, method: str) -> None:
+            local_error = None
+            try:
+                torch.cuda.synchronize()
+                if action == "sleep":
+                    release_with_tag(*tags)
+                    torch.cuda.synchronize()
+                    gc.collect()
+                    torch.cuda.empty_cache()
+                else:
+                    materialize_with_tag(*tags)
+                    torch.cuda.synchronize()
+            except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)